domain/src/test/kotlin/no/nav/su/se/bakover/domain/søknadsbehandling/StatusovergangTest.kt

```diff
@@ -513,7 +591,15 @@ internal class StatusovergangTest {
                 clock = fixedClock,
                 satsFactory = satsFactoryTestPåDato(),
                 nySaksbehandler = saksbehandler,
-            ).getOrFail() shouldBe beregnetInnvilget
+            ).getOrFail() shouldBe beregnetInnvilget.copy(
+                søknadsbehandlingsHistorikk = beregnetInnvilget.søknadsbehandlingsHistorikk.leggTilNyeHendelser(
+                    listOf(
+                        // Hvor og hvorfor oppdaterer den fradragene? :thinkies:
+                        nySøknadsbehandlingshendelse(handling = SøknadsbehandlingsHandling.OppdatertFradrag),
```

Comment:
Jeg tror vi bør legge inn denne handlingen slik atvi kun får en handling dersom saksbehandler utfører den. Jeg gjetter på at vi har lagt oss for langt inn.